model/validation.go

```diff
@@ -474,7 +475,7 @@ func validateCIDR(cidr string) error {
 
 	var errs error
 
-	if err := ValidateCIDRBlock(parts[1]); err != nil {
+	if err := ValidateCIDRBlock(cidr); err != nil {
```

Comment:
`net.ParseCIDR` calls `net.ParseIP`. Depending on whether we want to support IPv6 or not you could invoke `net.PraseCIDR` directly here and remove `validateCIDRBlock`

Reply:
Ack, I'm not sure if we specifically call out IPv6 support in Istio. It isn't 100% ready in the k8s sense, but it may be released in the alpha state for 1.9.